test-projects/test-database/build.sbt

```diff
@@ -0,0 +1,15 @@
+enablePlugins(DatabasePlugin)
+
+// You have to provide the database driver that corresponds to the JDBC URL
+libraryDependencies += org.allenai.plugins.CoreDependencies.postgresDriver
+
+// In order migratinos to work, you need to have postgres running
+// at the following URL, and have a database already created
+// named 'databaseplugin'
+flywayUrl := "jdbc:postgresql://localhost:5432/databaseplugin"
```

Comment:
Should this be an integration test then? I don't think this will work in Shippable?